src/features/abi-methods/models/index.ts

```diff
@@ -0,0 +1,54 @@
+export enum AbiType {
+  String = 'string',
```

Comment:
We mostly use PascalCase for the enum values elsewhere. Do the values need to be camel/lowercase?